.ci/ci

```diff
@@ -32,16 +32,16 @@ make coverage
 
 # Bootloader variants
 make -j8 bootloader
-make -j8 bootloader-devdevice
-make -j8 bootloader-devdevice-locked
+make -j8 bootloader-development
+make -j8 bootloader-development-locked
```

Comment:
kinda curious what's a locked devdevice supposed to do? I mean isnt the point of a dev device to be unlocked so newer FW versions can be flashed onto it?

Reply:
Locked does not refer to the signature checks on the firmware, but to MCU settings that prevent overwriting the flash with JLink.

Reply:
oh okay interesting to know, thanks